modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/display/context/ShippingMethodCheckoutStepDisplayContext.java

```diff
@@ -66,6 +78,36 @@ public CommerceOrder getCommerceOrder() {
 		return _commerceOrder;
 	}
 
+	public List<CommerceShippingFixedOption> getCommerceOrderFiltered()
```

Comment:
Why is this called getCommerceOrderFiltered? We are retrieveing CommerceShippingFixedOption not CommerceOrders